h2o-algos/src/main/java/hex/schemas/GLMV3.java

```diff
@@ -133,6 +134,9 @@
     @API(help = "Standardize numeric columns to have zero mean and unit variance", level = Level.critical)
     public boolean standardize;
 
+    @API(help = "If true will start GLM model from scratch, otherwise, will build from previous results", level = Level.critical)
```

Comment:
the doc needs improvements, every algo starts from scratch at the begging (unless checpoints are used), I guess this is related to the lambda search loop?

Reply:
That is correct.  Lambda search starts from old computationState values from last lambda value

Reply:
I changed the description.  Please take a look now.